fem/pfespace.hpp

```diff
@@ -428,6 +428,33 @@ class ParFiniteElementSpace : public FiniteElementSpace
       old_dof_offsets.DeleteAll();
    }
 
+   /** @brief Return the true-dof to true-dof update operator corresponding to
+       the last Mesh::Operation. */
+   /** This method is similar to the GetUpdateOperator() methods, however the
+       returned operator works on T-dof (true-dof) vectors instead of L-dof
+       vectors (i.e. ParGridFunction-sized vectors).
+
+       @param[in,out]  T  On input, the type of @a T is used as input to request
```

Comment:
How should an initial `T` be constructed, for the input? Can this function work without any input operator, just an input type?

Reply:
It is confusing to me, maybe because I haven't seen this function in typical usage, which might make things more clear. At first, it sounds tricky to need an operator that may not be constructed yet, just to get the type. It seems more natural to me just to input the type, if that is all this function needs. If there is a good reason to keep it the way it is (e.g. for convenience), then a comment should suffice, but I'm wondering if it is simpler just to input the type.